assets/js/base/components/checkout/form-step/style.scss

```diff
@@ -0,0 +1,69 @@
+$inner-circle-size: 24px;
+$circle-border-size: 8px;
+$outer-circle-size: $inner-circle-size + (2 * $circle-border-size);
+
+// @todo: replace those colors with the correct values once we settle on a design system palette.
+$gray-20: #c3c4c7;
```

Comment:
Let's move these colour definitions further up, e.g. in `abstracts` folder. This can be follow-up, not a blocker.

Reply:
> Let's move these colour definitions further up, e.g. in abstracts folder. This can be follow-up, not a blocker.

Agreed, I think this got mentioned in slack too?  Surfacing it in the `abstracts` folder will help with discovery later when we go to consolidate styles around one source.

Reply:
Yeah it was mentioned but I didn’t get to implement it until now